src/OrchardCore.Modules/OrchardCore.Google/Authentication/Configuration/GoogleOptionsConfiguration.cs

```diff
@@ -50,11 +56,13 @@ public void Configure(AuthenticationOptions options)
 
         public void Configure(string name, GoogleOptions options)
         {
-            if (!string.Equals(name, GoogleDefaults.AuthenticationScheme))
+            if (!String.Equals(name, GoogleDefaults.AuthenticationScheme))
             {
                 return;
             }
+
             var settings = GetGoogleAuthenticationSettingsAsync().GetAwaiter().GetResult();
+
             options.ClientId = settings?.ClientID ?? string.Empty;
```

Comment:
Here you have the same pattern than on the other class make it consistent.

Reply:
Good catch, i missed this one, thanks

Some options config classes have 2 `Configure()`, including this one for named options

Reply:
I will add just before, as done for others

            if (settings == null)
            {
                return;
            }

Without having to do a null check on `ClientID` afterwards